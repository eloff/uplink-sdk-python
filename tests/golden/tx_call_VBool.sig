r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 74743532417868210568748814056836691178813206007828783621706230588777052297970
