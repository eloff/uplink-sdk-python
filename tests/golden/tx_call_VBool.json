{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "contents": true, 
                        "tag": "VBool"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE03NDc0MzUzMjQxNzg2ODIxMDU2ODc0ODgxNDA1NjgzNjY5MTE3ODgxMzIwNjAwNzgyODc4MzYyMTcwNjIzMDU4ODc3NzA1MjI5Nzk3MA=="
}